docs/development/local_setup.md

```diff
@@ -69,31 +105,71 @@ $ git clone git@github.com:gardener/gardener.git
 $ cd gardener
 ```
 
-# Local development
+## Start the Gardener
 
 :warning: Before you start developing, please ensure to comply with the following requirements:
 
 1. You have understood the [principles of Kubernetes](https://kubernetes.io/docs/concepts/), and its [components](https://kubernetes.io/docs/concepts/overview/components/), what their purpose is and how they interact with each other.
 1. You have understood the [architecture of the Garden](https://github.com/gardener/documentation/wiki/Architecture), and what the various clusters are used for.
 
-The development of the Gardener could happen by targetting any cluster. You basically need a Garden cluster (e.g., a [Minikube](https://github.com/kubernetes/minikube) cluster) and one Seed cluster per cloud provider and per data center/region. You can configure the Gardener controller manager to watch **all namespaces** for Shoot manifests or to only watch **one single** namespace.
+The development of the Gardener could happen by targeting any cluster. You basically need a Garden cluster (e.g., a [Minikube](https://github.com/kubernetes/minikube) cluster) and one Seed cluster per cloud provider and per data center/region. You can configure the Gardener controller manager to watch **all namespaces** for Shoot manifests or to only watch **one single** namespace.
 
-In order to ensure that a specific Seed cluster will be chosen, add the `.spec.cloud.seed` field (see [here](../../example/shoot-azure.yaml#L10) for an example Shoot manifest).
+The commands bellow will set your `minikube` with the absolute minimum resources to launch Gardener API server & controller manager:
 
-## Get started
+```bash
+$ minikube start
+Starting local Kubernetes v1.8.0 cluster...
+[...]
+kubectl is now configured to use the kluster.
+
+$ make dev-setup
+clusterrolebinding "kube-dns-rule" created
+namespace "garden" created
+secret "default-domain" created
+secret "internal-domain-example-com" created
+deployment "etcd" created
+service "etcd" created
+```
```

Comment:
`endpoints "gardener-apiserver" created` is missing
`apiservice "v1beta1.garden.sapcloud.io" created` is missing